server/services/config/conditionsConfig.ts

```diff
@@ -45,4 +38,8 @@ export const multiFields = {
     fields: ['drug_testing_name', 'drug_testing_address'],
     joining: [', '],
   },
+  alcoholMonitoring: {
+    fields: ['timeframe', 'endDate'],
+    joining: [' ending on '],
```

Comment:
join the fields so in the text it says something like "weekly _ending on_ 12/09/22"